### ISSUE TITLE
[BUG] Check Link button closes program

### ISSUE THREAD

**Issue by stmorri** [Thatawesomeguy4/YoutubeDownloader#9]
**Describe the bug**

**To Reproduce**
When using the Check Link option, after I paste the URL the program closes with no error
I tried 2 URLs, the do actually download but for the link checker, try "https://www.youtube.com/watch?v=AwMLRXgvHW8&list=WL&index=34&t=40s" 
**Expected behavior**
Some type of confirmation message that the URL was valid.

**Screenshots**

**Desktop (please complete the following information):**
 Windows 10
 

**Additional context**

**Comment by Thatawesomeguy4** [comment]
Yeah, I investigated this and it looks like the check link code got deleted in one of my code merges. I'll work on getting that fixed and release a v1.0.1
